src/structures/GuildChannel.js

```diff
@@ -510,6 +510,10 @@ class GuildChannel extends Channel {
     if (this.client.user.id === this.guild.ownerId) return true;
     const permissions = this.permissionsFor(this.client.user);
     if (!permissions) return false;
+    // This flag allows managing even if timed out
```

Comment:
Nit:
```suggestion

    // This flag allows managing even if timed out
```

Reply:
Can admins even be timed out?

Reply:
Yes, yes they can

Reply:
Yes, as a matter of a fact, @vladfrangu timed me out in Sapphire, which has ADMINISTRATOR.

Reply:
You can still send messages with the API but not with the client if you have ADMINISTRATOR and you are timed out, and for I can see even if you are timed out in this scenario you can still connect to voice channels.